.travis.yml

```diff
@@ -1,16 +1,11 @@
 language: node_js
 sudo: false
 node_js:
-- '4'
 - '6'
 - '8'
 - '10'
 script:
-- npm run ci
-after_script:
-- npm install -g codeclimate-test-reporter
-- npm run cover
-- codeclimate-test-reporter < coverage/lcov.info
```

Comment:
Removing coverage?

Reply:
I noted in commit comment (https://github.com/apiaryio/fury-adapter-apib-serializer/pull/25/commits/626f13bb02d5b1866c4c5986a6d0f3f62aa7182a).

Since we're importing this library into the monorepo and we'll be doing it differently. As an interim solution I removed it due to it needing Peasant.

Reply:
👍